easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-3.0.10-gompi-2023a.eb

```
@@ -0,0 +1,73 @@
+# updated: Denis Kristak (INUITS)
+# updated: Sebastien Moretti (SIB - Vital-IT)
+# updated: Pavel Tománek (INUITS)
+easyblock = 'CMakeMake'
+
+name = 'SRA-Toolkit'
+version = '3.0.10'
+
+homepage = 'https://github.com/ncbi/sra-tools'
+description = """The SRA Toolkit, and the source-code SRA System Development
+ Kit (SDK), will allow you to programmatically access data housed within SRA
+ and convert it from the SRA format"""
+github_account = 'ncbi'
+
+toolchain = {'name': 'gompi', 'version': '2023a'}
+toolchainopts = {'extra_cflags': '-DH5_USE_110_API'}
+
+source_urls = ['https://github.com/ncbi/sra-tools/archive/refs/tags/']
+sources = [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['93cfa802938506866b9d565a18a31ac84fd26f2929636b23f1f8dd9f39cf977d']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.8.2'),
+    ('Perl', '5.36.1'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Java', '11', '', SYSTEM),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('ncbi-vdb', version),
+    ('bzip2', '1.0.8'),
+    ('file', '5.43'),
+    ('HDF5', '1.14.0'),
+    ('libxml2', '2.11.4'),
+    ('zlib', '1.2.13'),
+]
+
+configopts = '-DVDB_INCDIR="$EBROOTNCBIMINVDB/include" -DVDB_LIBDIR="$EBROOTNCBIMINVDB/lib" '
+configopts += '-DBUILD_TOOLS_LOADERS=ON -DBUILD_TOOLS_INTERNAL=ON'
+
+postinstallcmds = [
+    "cp -r %(builddir)s/sra-tools-3.0.10/ngs/ngs-python/ %(installdir)s/",
```

Let's avoid hardcoding the version number here, since that's going to be trouble with future updates:

```suggestion
    "cp -r %(startdir)s/ngs/ngs-python/ %(installdir)s/",
```